src/OSPSuite.Presentation/Services/ExceptionManager.cs

```diff
@@ -27,7 +28,11 @@ public ExceptionManager(IDialogCreator dialogCreator, IExceptionView exceptionVi
 
       public override void LogException(Exception ex)
       {
-         if (ex.IsInfoException())
+         if (ex is CancelCommandRunException)
+         {
+            // empty clause here because we simply are unwinding the call stack
```

Comment:
the plan is to throw exceptions when users decline to convert their pk-sim modules to extension modules. We are just going to unwind the stack when they choose to cancel. That prevents other commands from running if they are in a sequence